third_party/java/android_databinding/v3_4_0/BUILD

```diff
@@ -13,60 +13,10 @@ filegroup(
     srcs = glob(["**"]),
 )
 
-# Base library shared by build tools and the data binding runtime.
-java_import(
-    name = "base",
-    jars = ["base.jar"],
-)
-
 # Resource processor implementation.
+# Built from https://cs.android.com/androidx/platform/frameworks/data-binding/+/mirror-goog-studio-master-dev:exec/src
```

Comment:
just curious, so this thing isn't available on maven? are there plans to put it on maven?

Reply:
ah just saw the PR description.

Reply:
It's a custom tool built for Bazel, and the source is in kotlin, so hopefully it will just be removed from the Bazel repo when we move out android tools stuff after starlarkifying the android rules.